docs/exposing_dns.md

```diff
@@ -40,18 +40,24 @@ kubectl create -n ingress-nginx cm udp-services --from-literal="53"="k8gb/k8gb-c
 
 [Local project setup](./local.md) does this patching automatically.
 
+## CoreDNS service lookup
+k8gb no longer hardcodes coredns LB service name, instead it is trying to find a service annotated with `app.kubernetes.io/name=coredns` within the same namespace where controller itself is deployed into. If service has `Status.LoadBalancer.Ingress[0].Hostname`, k8gb will use the resolved IPs to reach CoreDNS on this cluster.
```

Comment:
I think we should not overload the documentation with the historical approach( which is good for commit message but not a doc). 
```suggestion
k8gb is trying to find a service annotated with `app.kubernetes.io/name=coredns` within the same namespace where the controller itself is deployed into. If the service has `Status.LoadBalancer.Ingress[0].Hostname`, k8gb will use the resolved IPs to reach CoreDNS on this cluster.
```

Reply:
fixed